### DISCUSSION TITLE
All Cell voltages in VRM

### DISCUSSION THREAD

**Discussion by RFDarter** [mr-manuel/venus-os_dbus-mqtt-battery#18] [Q&A]
Hi,
All cell voltages are transmitted to the dbus.
But in VRM i can only display the min and Max cell.
Ist that a limitation of VRM or do I need to aktivate something?

**Comment by mr-manuel** [comment]
This is a limitation of VRM. Originally you don't even see the cell voltages in Venus OS. Only this driver is providing the single cell voltages.

**Comment by mhin76** [comment]
interesting enough it did show all cells last week remote console. But now that option disappeared again (even I did not change anything).  
here I had an entry called cell voltages:
![image](https://github.com/user-attachments/assets/ce744202-e385-4b28-b43b-ab1b25ee253f)
